action.yml

```diff
@@ -109,6 +109,14 @@ inputs:
         proxy_servers:
           http: ...
       ```
+  post-deinit:
+    description: |-
+      Whether the micromamba installation should be deinitialized in the post action of the workflow.
+      This is useful for self-hosted runners that keep the state of the system.
+      `micromamba deinit` is implemented for `micromamba>=0.25.0` so this must be set to `false` 
+      if you want to use an older `micromamba` installation.
+    required: true
+    default: true
```

Comment:
Can we change this to default to `false` for versions that don't support `deinit`? Otherwise we will break setups with a `micromamba-version: X`, `X<0.25` pin. For simplicity, we can just assume that we have a version >=0.25 unless we find `micromamba-version: X` in the args, with `X<0.25`. Alternatively we can also run `micromamba shell --help` and look for `"deinit"`.

Reply:
So let's do `default: auto` with your described behavior.